pootle/static/js/browser.js

```diff
@@ -175,37 +175,50 @@ function fixDropdowns() {
 }
 
 
+function formatResult(text, term) {
+  const match = text.toUpperCase().indexOf(term.toUpperCase());
+  const tl = term.length;
+  const markup = [];
+
+  if (match < 0) {
+    markup.push(_.escape(text));
```

Comment:
Also since `markup` is going to be changed perhaps it is better to define it with `let`.